snowflake_paramstore.c

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
I assume this is just copied over from libsnowflakeclient

Reply:
Mostly yes. The Data Structures are copied. Slight changes to some functions and return types. Most new changes are in snowflake_stmt.c